code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -0,0 +1,55 @@
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/silence
+	name = "The Price of Silence"
+	desc = "A scythe with a clock attached, quitely ticking."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "silence"
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 40, 40),
+		ABNORMALITY_WORK_INSIGHT = 0,
+		ABNORMALITY_WORK_ATTACHMENT = list(0, 0, 10, 20, 30),
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 45, 45)
+			)
+	start_qliphoth = 1
+	work_damage_amount = 10
+	work_damage_type = PALE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/thirteen,
+		/datum/ego_datum/armor/thirteen,
+		)
+//	gift_type = /datum/ego_gifts/thirteen
+
+	var/meltdown_cooldown_time = 780 SECONDS	//13 minutes
+	var/meltdown_cooldown
+	var/worldwide_damage = 70	//If you're unarmored, it obliterates you
+	var/safe = FALSE //work on it and you're safe for 13 minutes
+
+
+/mob/living/simple_animal/hostile/abnormality/silence/Life()
+	. = ..()
+	playsound(get_turf(src), 'sound/abnormalities/silence/bong.ogg', 100, 0, 100)	//Church bells ringing, whether it happens or not.
```

Comment:
Just use sound for entire z level proc instead of setting range to 100.
Additionally, you are playing the sound without any checks every tick(2 seconds).